src/libraries/System.Configuration.ConfigurationManager/ref/System.Configuration.ConfigurationManager.cs

```diff
@@ -1578,6 +1578,15 @@ public partial class ProviderException : System.Exception
         public ProviderException(string message, System.Exception innerException) { }
     }
 }
+#if NET7_0_OR_GREATER
+namespace System.Diagnostics
+{
+    public static partial class TraceConfiguration
+    {
+        public static void Register() { }
```

Comment:
Did you guys settle on Register() as the name? I still don't care for it but if I'm the odd one out no worries.

Reply:
It was brought up, and there were some suggestions in the issue including your recent comment, but no real pushback on "Register()". Since the method lives in the new `TraceConfiguration` static class, and it's the only method there, that helps with finding it at least.

https://github.com/dotnet/runtime/issues/72967#issuecomment-1198444939
https://github.com/dotnet/runtime/issues/72967#issuecomment-1208910234

Reply:
My only concern here is misalignment of vocab - we use "register" for the method but event args use "initializing".

Reply:
I'll follow-up on this with @ericstj

Reply:
IMO both names "do the job".  WRT the difference in "register" and "initializing" I see a difference.  "Initializing" for event args refers to initializing new objects, where as "register" refers to attaching the event handlers to do that "initialization" work later on.  

A couple other places we've used `Register` when separating technologies like this:
https://docs.microsoft.com/en-us/dotnet/api/system.text.encoding.registerprovider?view=net-6.0
https://docs.microsoft.com/en-us/dotnet/api/system.net.authenticationmanager.register?view=net-6.0
https://docs.microsoft.com/en-us/dotnet/api/system.uriparser.register?view=net-6.0
There are quite a few more.